src/app/welcome-page/welcome-page.component.html

```diff
@@ -54,12 +54,15 @@ <h2>Document the evidence, and unmask the trolls</h2>
         Use SafeNet to flag abusive content, report the abuse to the platforms or get our forensic investigators to
         track the culprits.
       </p>
-      <button mat-raised-button class="twitter-button">
-        <span>
-          <mat-icon class="product-icon">visibility</mat-icon>
-          Find out how
-        </span>
-      </button>
+      <a href="/help-center">
```

Comment:
I think [we shouldn't nest interactive content](https://stackoverflow.com/a/6393863/5562041) inside the `<a>` tag